common4j/src/main/com/microsoft/identity/common/java/controllers/BaseController.java

```diff
@@ -306,7 +306,8 @@ protected final AuthorizationRequest.Builder initializeAuthorizationRequestBuild
             completeRequestHeaders.put(AuthenticationConstants.AAD.APP_VERSION,
                     parameters.getApplicationVersion()
             );
-            completeRequestHeaders.put(PKEYAUTH_HEADER, PKEYAUTH_VERSION);
```

Comment:
we should still send this as this is going to the auth endpoint, not token endpoint.